packages/facebook/facebook_client.js

```diff
@@ -35,7 +35,12 @@ Facebook.requestCredential = function (options, credentialRequestCompleteCallbac
         '&redirect_uri=' + OAuth._redirectUri('facebook', config) +
         '&display=' + display + '&scope=' + scope +
         '&state=' + OAuth._stateParam(loginStyle, credentialToken, options && options.redirectUrl);
-
+  
+  // Handle authentication type (e.g. for force login you need authType: "reauthenticate")
+  if (options.authType) {
+    loginUrl = loginUrl + "&authType=" + options.authType;
```

Comment:
@benjamn I don't think that's really needed, but just in case, I just pushed it.